dasharo-compatibility/me-dock-comp.robot

```diff
@@ -0,0 +1,114 @@
+*** Settings ***
+Library             Collections
+Library             OperatingSystem
+Library             Process
+Library             String
+Library             Telnet    timeout=20 seconds    connection_timeout=120 seconds
+Library             SSHLibrary    timeout=90 seconds
+Library             RequestsLibrary
+Resource            ../variables.robot
+Resource            ../keywords.robot
+Resource            ../keys.robot
+Resource            ../pikvm-rest-api/pikvm_comm.robot
+Resource            ../lib/me.robot
+
+# TODO:
+# - document which setup/teardown keywords to use and what are they doing
+# - go threough them and make sure they are doing what the name suggest (not
+# exactly the case right now)
+Suite Setup         Run Keywords
+...                     Prepare Test Suite
+...                     AND
+...                     Skip If    not ${DASHARO_INTEL_ME_MENU_SUPPORT}    Dasharo Intel ME menu not supported
+...                     AND
+...                     Set UEFI Option    MeMode    Enabled
+Suite Teardown      Run Keyword
+...                     Log Out And Close Connection
+
+
+*** Test Cases ***
+TMD004.001 USB Type-C Display output With Me Disabled(semi-automatic)
```

Comment:
Any chance to put these in the same file as the existing USB-C / thunderbolt tests? Maybe as a variant with suffix `(ME disabled)`

ideally the regular and ME disabled cases would prepare the ME state call a common keyword to do the test

Reply:
ok how about that : https://github.com/Dasharo/open-source-firmware-validation/pull/488/commits/eeda739962e40f269941ae962e34ddb0110c76b1

Reply:
That's better, thanks.